saturn-pytorch/recipe-template.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "saturn-pytorch",
+    "description": "An image for deep learning with PyTorch. Includes all requirements for using PyTorch on a GPU and with Dask.",
```

This is stylistic, but do we really want to start all these with "An image for..."? This feels snappier to me:

```suggestion
    "description": "For deep learning with PyTorch. Includes all requirements for using PyTorch on a GPU and with Dask.",
```